scavenger-collector/src/main/kotlin/com/navercorp/scavenger/service/GarbageCollectService.kt

```diff
@@ -101,8 +101,9 @@ class GarbageCollectService(
                     customerId,
                     baseDateTime.minusMillis(intervalService.batchSweepMarginMilliSecond)
                 )
+                val jvmUuids = jvmDao.findAllUuidsByWithoutAgent(customerId)
```

Comment:
How about `uuidsWithoutAgent` instead of `jvmUuids`?

Reply:
Doesn't a method name that explicitly states "find UUIDs without an agentState" seem clearer?

Reply:
> How about `uuidsWithoutAgent` instead of `jvmUuids`?

Doesn't a method name that explicitly states "find UUIDs without an agentState" seem clearer?